submissions/TatyanaMolchanova/js-dom/script.js

```diff
@@ -0,0 +1,103 @@
+const content = {
+    'cats': {
```

Comment:
You can't change this but next time it's better to make your data simpler and more dynamic.
Your data would be an array like this:
```javascript
const animalsData = [
 {
 menuTitle: 'Cats',
 class: 'cats',
 title: 'Cats are awesome',
 info: 'Having a cat can mean different things to different people. Some want a cat to cuddle and sit on their laps; others are happy to live with a very independent cat which spends most of its time outside and doesn’t want too much human interaction.',
 image: 'cat'
 }, {...}, {...}
]
```
It helps you easily use `Array.prototype` methods and make your code more flexible to the incoming data. In your decision you strongly depend on the incoming data and any of the changes in the data object can break the whole app. Try to think about this in future tasks.

Reply:
Ok, thank you! I'll try to use as a key not a string.